src/index.ts

```diff
@@ -4,6 +4,8 @@ import through from 'through';
 import ts, { factory } from 'typescript';
 
 import browserify from '@cypress/browserify-preprocessor';
+// @ts-ignore:next-line
```

Comment:
Instead of ignoring the next line you can define the module  in `index.d.ts` similar to what we do for `@cypress/browserify-preprocessor`.

```
declare module 'boolean-parser';
```